src/System.Security.Cryptography.Csp/tests/System.Security.Cryptography.Csp.Tests.csproj

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
> Likewise, presumably this line should be deleted?

Yes, I didn't see them.  They should be removed.

> (Also, shouldn't each of the changes be down here, after the dir.props import?)

I'm just following what's done in other platform-specific .csproj tests, e.g.
https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.OpenSsl/tests/System.Security.Cryptography.OpenSsl.Tests.csproj#L3-L6
https://github.com/dotnet/corefx/blob/master/src/System.Security.Principal.Windows/tests/System.Security.Principal.Windows.Tests.csproj#L3-L6
etc.